controller/README.md

```diff
@@ -0,0 +1,3 @@
+Controllers are called by the Codec Gorilla component and marshal/unmarshal requests and responses to functions in the gaeway package. There is one Controller per supported method or namespace.
+
+Functions in the gateway package actually implement that method or namespace's functionality.
```

Comment:
I would add: Functions in the gateway package implement the bussines rules, meaning all the use cases

Reply:
Fixed